source/_integrations/unifi.markdown

```diff
@@ -112,14 +112,11 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
-These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.
+These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks. The integration only supports per-device sensors.
 
-1. To enable the sensors, on the UniFi integration page, select **Configure**. 
+1. To enable the sensors, on the UniFi integration page, select **Configure**.
 2. Go to page 3/3 and enable the sensors.
```

Comment:
Remove this

Reply:
The reason I changed this is that for procedural content, we use numbered procedures. But I can see that this is an edge case that can also work as a one-sentence instruction.

Reply:
It's only true for two sensors as well. Newer sensors aren't offered any configuration